lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/oauth/OAuthClientAuthorizationHistory.json

```diff
@@ -0,0 +1,25 @@
+{
+    "description": "An object representing an OAuth 2.0 client that has been authorized to request a user's resources and identity",
+ 	"properties": {
+		"client": {
+			"$ref": "org.sagebionetworks.repo.model.oauth.OAuthClient",
+			"description": "The OAuth 2.0 client that has been granted access",
+			"links": [
```

Comment:
you can omit 'links' and the service that uses this object should return ONLY the public client fields.